semgrep-core/src/parsing/tree_sitter/Parse_swift_tree_sitter.ml

```diff
@@ -908,18 +1095,24 @@ and map_block (env : env) ((v1, v2, v3) : CST.block) =
 
 and map_bodyless_function_declaration (env : env) ~in_class
     ((v1, v2, v3) : CST.bodyless_function_declaration) body =
-  let v1 =
+  let attrs =
     match v1 with
-    | Some x -> map_modifiers env x |> todo env
-    | None -> ()
+    | Some x -> map_modifiers env x
+    | None -> []
   in
+  (* I'm not sure this does anything. "class" is already a possible modifier
+     from the `map_modifiers` - it's a property modifier.
+  *)
   let v2 =
     match v2 with
-    | Some tok -> (* "class" *) token env tok |> todo env
+    | Some tok ->
+        (* "class" *)
+        let _todo = token env tok in
```

Comment:
Pipe this to `todo env` so that it doesn't get overlooked if we do encounter it?